contracts/tokens/SecurityToken.sol

```diff
@@ -84,6 +85,17 @@ contract SecurityToken is ERC20, ERC20Detailed, Ownable, ReentrancyGuard, Securi
             require(_isModule(msg.sender, _type));
     }
 
+    function _isValidPartition(bytes32 _partition) internal pure {
+        require(_partition == UNLOCKED, "Invalid partition");
```

Comment:
Can it not also be LOCKED?

Reply:
Ignore comment - I see you're only using this for transfers / operators etc., you can still call balanceOfPartition on LOCKED.